resources/views/torrent/results.blade.php

```diff
@@ -143,13 +143,29 @@ class="torrent-poster-img-small show-poster" alt="@lang('torrent.poster')">
                                     <i class="{{ config('other.font-awesome') }} fa-download"></i>
                                 </button>
                             </a>
+                            @if (config('torrent.magnet') == 1)
```

Comment:
Is this supposed to be here twice?

Reply:
Yeah this was a silly mistake on my end. I'll clean it up. ;)